File: src/main/java/gr/apt/repository/notification/NotificationRecipientPersonsRepository.java
package gr.apt.repository.notification;

import gr.apt.persistence.entity.notification.NotificationRecipientPersons;
import io.quarkus.hibernate.orm.panache.PanacheRepositoryBase;

import javax.enterprise.context.ApplicationScoped;
import java.math.BigInteger;
@ApplicationScoped
public class NotificationRecipientPersonsRepository implements PanacheRepositoryBase<NotificationRecipientPersons, BigInteger> {
}

File: src/main/java/gr/apt/repository/notification/NotificationRecipientPersonsRepository.kt
package gr.apt.repository.notification

import gr.apt.persistence.entity.notification.NotificationRecipientPersons
import io.quarkus.hibernate.orm.panache.PanacheRepositoryBase
import java.math.BigInteger
import javax.enterprise.context.ApplicationScoped

@ApplicationScoped
class NotificationRecipientPersonsRepository : PanacheRepositoryBase<NotificationRecipientPersons?, BigInteger?>